From: Sophie Brun <sophie@offensive-security.com>
Date: Mon, 18 Jan 2021 11:38:24 +0700
Subject: Adapt service file with Kali files localization

Last-Update: 2019-03-04

Last-Update: 2019-03-04
---
 bettercap.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bettercap.service b/bettercap.service
index 82bc642..23cc4fb 100644
--- a/bettercap.service
+++ b/bettercap.service
@@ -7,7 +7,7 @@ After=network.target
 [Service]
 Type=simple
 PermissionsStartOnly=true
-ExecStart=/usr/local/bin/bettercap -no-colors -eval "set events.stream.output /var/log/bettercap.log; api.rest on"
+ExecStart=/usr/bin/bettercap -no-colors -eval "set events.stream.output /var/log/bettercap.log; api.rest on"
 Restart=always
 RestartSec=30
 
