Description: Disable service autostart
Author: Lorenzo "Palinuro" Faletra <palinuro@parrotsec.org>
Last-Update: 2019-03-29

--- bettercap-2.19.orig/bettercap.service
+++ bettercap-2.19/bettercap.service
@@ -10,6 +10,3 @@ PermissionsStartOnly=true
 ExecStart=/usr/bin/bettercap -no-colors -eval "set events.stream.output /var/log/bettercap.log; api.rest on"
 Restart=always
 RestartSec=30
-
-[Install]
-WantedBy=multi-user.target
